src/sg/gov/moh/pdt-healthcert/2.0/schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemata.openattestation.com/sg/gov/moh/pdt-healthcert/2.0#",
+  "type": "object",
+  "required": ["id", "version", "type", "validFrom", "fhirVersion", "fhirBundle"],
+  "properties": {
+    "id": {
+      "type": "string",
+      "examples": ["TEST001"]
+    },
+    "version": {
+      "type": "string",
+      "enum": ["PDT-HealthCert-v2.0"]
```

Comment:
I'm personnaly not verry found of fixed values that mixes cases. I suggest: "pdt-healthcert-v2.0"